package.json

```diff
@@ -93,8 +93,8 @@
     "react-onclickoutside": "^6.13.0",
     "react-redux": "7.2.9",
     "react-responsive": "9.0.2",
-    "react-router": "6.23.1",
-    "react-router-dom": "6.23.1",
+    "react-router": "6.27.0",
+    "react-router-dom": "6.27.0",
```

Comment:
Is this intended?
Not a problem, though.

Reply:
I was checking if `useBlocker` is fixed in the latest version and then left it as is.